src/lib.rs

```diff
@@ -49,13 +49,10 @@ extern crate alloc;
 #[cfg(feature = "no-std")]
 extern crate core2;
 
-#[cfg(any(feature = "std", test))]
-extern crate core; // for Rust 1.29 and no-std tests
-
 // Re-exported dependencies.
 #[macro_use] pub extern crate bitcoin_hashes as hashes;
-pub extern crate secp256k1;
-pub extern crate bech32;
+pub use secp256k1;
+pub use bech32;
```

Comment:
Nit: I still think that doing `pub extern crate` is better in this case since it helps to clearly see that this two are not local modules co-named with other crate, but an external crate